app/services/obituary_service.rb

```diff
@@ -0,0 +1,26 @@
+class ObituaryService
+
+  def get_obituaries
+    JSON.parse(conn.get.body, symbolize_names: true)
+  end
+
+  def get_covid_obituaries
+    resp = conn.get('/covid')
+    JSON.parse(resp.body, symbolize_names: true)
+  end
+
+  def get_obituaries_by_name(name)
+    resp = conn.get('/name') do |req|
+      req.params['name'] = "#{name}"
+    end
+    JSON.parse(resp.body, symbolize_names: true)
+  end
```

Comment:
will this search for any name match? I'm not sure how the microservice is working. Maybe you and Jack can walk us through that this afternoon.

Reply:
Good question Stella. Curious to know if it only takes letters, if it sterilizes special characters, also how it does with spaces?